packages/manager-react-components/src/components/badge/index.ts

```diff
@@ -0,0 +1 @@
+export { Badge as BadgeComponent } from './badge.component';
```

Comment:
wouldn't that mean that you have to use it like `<BadgeComponent />` instead of `<Badge />` ?